src/components/InternalOverlay/pageObject/button.js

```diff
@@ -19,6 +19,15 @@ class PageInternalOverlayButton {
     click() {
         $(this.rootElement).click();
     }
+
+    /**
+     * Returns true when the trigger element has focus.
+     * @method
+     * @returns {bool}
+     */
+    hasFocusTrigger() {
+        return $(this.rootElement).isFocused();
+    }
```

Comment:
we were wrong with this page object, I think we should remove it since the trigger element is not part of the InternalOverlay, instead simply get the trigger button and do whatever needed in the test